ember-cli-build.js

```diff
@@ -14,5 +14,6 @@ module.exports = function(defaults) {
     behave. You most likely want to be modifying `./index.js` or app's build file
   */
 
-  return app.toTree();
+  const { maybeEmbroider } = require('@embroider/test-setup');
+  return maybeEmbroider(app);
```

Comment:
```suggestion
  return maybeEmbroider(app, {
    skipBabel: [
      { package: 'qunit' },
    ],
  });
```

This should fix the CI failures in embroider scenarios

Reply:
Thanks!

Reply:
Eeck, sorry @rwwagner90. Looks like my inline suggestion broke prettier :(

Mind running `yarn lint:fix`?